schema/deploy/shipping/views.sql

```diff
@@ -2083,6 +2084,46 @@ comment on view shipping.flu_assembly_jobs_v1 is
 alter view shipping.flu_assembly_jobs_v1 owner to "view-owner";
 
 
+create or replace view shipping.genome_submission_metadata_v1 as
+
+    select
+        sample.identifier as sfs_sample_identifier,
+        sampleid.barcode as sfs_sample_barcode,
+        sample.collected::date as collection_date,
+        case sample.details ->> 'swab_type'
+          when 'ans' then 'Anterior nasal swab'
+          when 'mtb' then 'Mid-turbinate nasal swab'
+          when 'np' then 'Nasopharyngeal swab'
+          else null
+        end as swab_type,
+        case
+          when identifier_set.name in ('collections-scan', 'collections-scan-kiosks') then 'SCAN'
+          else 'SFS'
```

Comment:
If not SCAN then SFS ... this won't hold forever. Apple, for example. 
There's also discussion about us not including Apple data with other data. That's still being discussed, so we don't have to filter anything just yet. I can imagine having to exclude the two Apple collection sets from this view.

Reply:
SFS/SCAN is the org, not the study. I think all current and future studies are happening under the aegis of SFS.

Reply:
Everything is still under the SFS org, the separation of SCAN is to add extra authors from PHSKC. 
I've asked Trevor on [Slack](https://seattle-flu-study.slack.com/archives/C01MQTSJH44/p1614717832026400) to verify that all other SFS studies can use the general SFS authors list.